flaskdeid/annotation.py

```diff
@@ -1,15 +1,24 @@
 """Module for standardizing and combining annotations"""
+from operator import attrgetter
 
 # TODO: If more origins/mappings added, rip this out into separate json file
```

Comment:
+1
I'd rather see this in some kind of separate config. but it's cool where it is for now